auth/store.go

```diff
@@ -997,18 +997,23 @@ func (as *authStore) AuthInfoFromCtx(ctx context.Context) (*AuthInfo, error) {
 		return nil, nil
 	}
 
-	ts, tok := md["authorization"]
-	if !tok {
-		return nil, nil
-	}
-
-	token := ts[0]
-	authInfo, uok := as.authInfoFromToken(ctx, token)
-	if !uok {
-		plog.Warningf("invalid auth token: %s", token)
-		return nil, ErrInvalidAuthToken
+	for k := range md {
+		if k == "token" || k == "authorization" {
```

Comment:
@well grpc-gateway for now only allows direct access to authorization header.  This may change if you follow issue.  Otherwise  I think that token could be used if you passed header `Grpc-Metadata-Token` but this is not very intuitive. https://github.com/grpc-ecosystem/grpc-gateway/issues/311

So perhaps time will resolve this question.